src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectPropertyDescriptor.cs

```diff
@@ -635,14 +534,15 @@ public override void AddValueChanged(object component, EventHandler handler)
 
             // If there's an event called <propertyname>Changed, hook the caller's handler directly up to that on the component
             EventDescriptor changedEvent = ChangedEventValue;
-            if (changedEvent != null && changedEvent.EventType.IsInstanceOfType(handler))
+            if (changedEvent != null && changedEvent.EventType.GetTypeInfo().IsInstanceOfType(handler))
             {
                 changedEvent.AddEventHandler(component, handler);
             }
 
             // Otherwise let the base class add the handler to its ValueChanged event for this component
             else
             {
+#if FEATURE_PROPERTYCHANGED
```

Comment:
We have `FEATURE_PROPERTY_CHANGED_EVENT_HANDLER` below; should these be the same?